templates/en/codelist.rst

```diff
@@ -9,10 +9,10 @@
 External URL: {{codelist_json.metadata.url}}
 {% endif %}
 
-{% if embedded %}
```

Comment:
As per [this review](https://github.com/IATI/IATI-Standard-SSOT/pull/309#pullrequestreview-229561208), just remove the template variable from `gen.py`